<launch>

	<!-- Torque-based PID -->
	<arg name="p_gain" default="15"/>
	<arg name="i_gain" default="0"/>
	<arg name="d_gain" default="2"/>

	<node name="pid_torque_jaco" pkg="iact_control" type="pid_torque_jaco.py" args="$(arg p_gain) $(arg i_gain) $(arg d_gain)" output="screen"/> 

	<!-- Visualization -->
	<node name="rvizualizer" pkg="iact_control" type="rvizualizer.py"/> 

	<!-- Start RVIZ --> 
	<node pkg="rviz" type="rviz" name="rviz"/>

</launch>
